packages/EasyCodingStandard/src/config/config.yml

```diff
@@ -0,0 +1,7 @@
+imports:
+    - { resource: '../../packages/**/src/config/services.yml' }
+    - { resource: 'services.yml' }
+    - { resource: 'external-services.yml' }
+
+parameters:
+    cache_directory: '%kernel.cache_dir%'
```

Comment:
Maybe a subdirectory for ChangedFilesDetector cache would be nice?

Reply:
Sure